src/Extensions/Hosting/Enviroments.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using Microsoft.Extensions.Hosting;
+
+namespace Microsoft.Omex.Extensions.Hosting
+{
+	/// <summary>
+	/// Provides variables with commonly used environments
+	/// </summary>
+	public static class OmexEnviroments
```

Comment:
>OmexEnviroments [](start = 21, length = 15)

Typo OmexEnvironments -- i.e. missing n. Same for the filename, which should actually be OmexEnvironments.cs as the Omex is completely missing.

Reply:
Renamed, thank you